snmp/datadog_checks/snmp/data/profiles/cisco.yaml

```diff
@@ -0,0 +1,9 @@
+# Profile for Cisco  devices
+
+extends:
+  - _base_cisco.yaml
```

Comment:
If we have a base cisco profile, this is arguably an extension of that so why do we define the sysobjectid here rather than in the `_base_cisco.yaml`?

Reply:
`_base_cisco.yaml` is an abstract profile intended to be extended by other cisco profiles which is why we don;t define any sysoids in it